api/logics.py

```diff
@@ -1009,10 +1009,17 @@ def cancel_order(cls, order, user, state=None):
         if order.status in do_not_cancel:
             return False, {"bad_request": "You cannot cancel this order"}
 
-        # 1) When maker cancels before bond
+        # 1) If the order status is not the one specified by the user, do not cancel the order.
```

Comment:
Best to leave this "do not cancel` comments without enumeration. That is, remove the "1)" , and leave all other cases with the number they had. This is consistent with the first cancel rejection case just above this one:
```
        if order.status in do_not_cancel:
            return False, {"bad_request": "You cannot cancel this order"}
```
The intention of enumerating was to have a clear map of the different cancellation conditions and bond slashing because they are all unique in some way.